docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+
+services:
+    csv-generator:
+        build:
+            context: ./csv_generator
+        command: '--help'
```

Comment:
not sure if there is an instruction not to start the service

Reply:
good default